sdks/python/src/opik/integrations/langchain/opik_tracer.py

```diff
@@ -344,6 +349,65 @@ def _on_llm_start(self, run: "Run") -> None:
 
         self._process_start_span(run)
 
+    def on_chat_model_start(
+        self,
+        serialized: dict[str, Any],
+        messages: list[list["BaseMessage"]],
+        *,
+        run_id: "UUID",
+        tags: Optional[list[str]] = None,
+        parent_run_id: Optional["UUID"] = None,
+        metadata: Optional[dict[str, Any]] = None,
+        name: Optional[str] = None,
+        **kwargs: Any,
+    ) -> "Run":
+        """Start a trace for an LLM run.
+
+        Duplicated from Langchain tracer, it is disabled by default in all tracers, see https://github.com/langchain-ai/langchain/blob/fdda1aaea14b257845a19023e8af5e20140ec9fe/libs/core/langchain_core/callbacks/manager.py#L270-L289 and https://github.com/langchain-ai/langchain/blob/fdda1aaea14b257845a19023e8af5e20140ec9fe/libs/core/langchain_core/tracers/core.py#L168-L180
+
+        Args:
+            serialized: The serialized model.
+            messages: The messages.
+            run_id: The run ID.
+            tags: The tags. Defaults to None.
+            parent_run_id: The parent run ID. Defaults to None.
+            metadata: The metadata. Defaults to None.
+            name: The name. Defaults to None.
+            kwargs: Additional keyword arguments.
+
+        Returns:
+            Run: The run.
+        """
+        start_time = datetime.now(timezone.utc)
+        if metadata:
+            kwargs.update({"metadata": metadata})
+
+        # We switched from langchain dumpd to model_dump() as we don't need all the langchain stuff
+        chat_model_run = Run(
+            id=run_id,
+            parent_run_id=parent_run_id,
+            serialized=serialized,
+            inputs={
+                "messages": [[msg.model_dump() for msg in batch] for batch in messages]
+            },
+            extra=kwargs,
+            events=[{"name": "start", "time": start_time}],
+            start_time=start_time,
+            run_type="llm",
+            tags=tags,
+            name=name,  # type: ignore[arg-type]
+        )
+        self._start_trace(chat_model_run)
```

Comment:
let's maybe use `super()` syntax here to highlight that it is a superclass method.

Reply:
I mostly copied it from Langchain tracer https://github.com/langchain-ai/langchain/blob/fdda1aaea14b257845a19023e8af5e20140ec9fe/libs/core/langchain_core/tracers/langchain.py#L128-L172. The super method is doing too much for us and rely on the problematic method `_create_chat_model_run` https://github.com/langchain-ai/langchain/blob/master/libs/core/langchain_core/tracers/base.py#L54-L93 which contains the default "fallback" logic for tracer. So I don't think that calling the `super()` class is needed nor useful in this case.
However the super class also has an async version of this method: https://github.com/langchain-ai/langchain/blob/master/libs/core/langchain_core/tracers/base.py#L572, I will check if I need to override it also